LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2020, Simon Billinge
```

Comment:
I think we should change this to copyright Columbia University...please find another license that has the correct wording for this (xpdAcq maybe?)